docs/how-to-guides/upgrade-rook-ceph.md

```diff
@@ -0,0 +1,165 @@
+# Upgrading Rook Ceph
+
+## Contents
+
+- [Introduction](#introduction)
+- [Steps](#steps)
+  - [Step 1: Make a note of existing image versions](#step-1-make-a-note-of-existing-image-versions)
+  - [Step 2: Ensure autoscale is on](#step-2-ensure-autoscale-is-on)
+  - [Step 3: Watch ceph status](#step-3-watch-ceph-status)
+  - [Step 4: Watch pods in rook namespace](#step-4-watch-pods-in-rook-namespace)
+  - [Step 5: Watch rook version update](#step-5-watch-rook-version-update)
+  - [Step 6: Watch ceph version update](#step-6-watch-ceph-version-update)
+  - [Step 7: Watch events in rook namespace](#step-7-watch-events-in-rook-namespace)
+  - [Step 8: Ceph dashboard](#step-8-ceph-dashboard)
+  - [Step 9: Grafana dashboard](#step-9-grafana-dashboard)
+  - [Step 10: Perform updates](#step-10-perform-updates)
+  - [Step 11: Verify that the CSI images are updated](#step-11-verify-that-the-csi-images-are-updated)
+  - [Step 12: Final checks](#step-12-final-checks)
+- [Additional resources](#additional-resources)
+
+## Introduction
+
+Rook Ceph is one of the storage providers of Lokomotive. With a distributed system as complex as
+Ceph, the upgrade can be tricky. This document enlists steps on how to keep an eye on the upgrade
+process.
+
+## Steps
+
+Following steps are inspired from [`rook`](https://rook.io/docs/rook/master/ceph-upgrade.html) docs.
+
+### Step 1: Make a note of existing image versions
```

Comment:
First thing first, I'd ensure that cluster is in the healthy state.

Reply:
I have made some changes, PTAL. In the `ceph status` there is an important note which says don't move forward if there is anything other than `HEALTH_OK`.

Reply:
I meant, before starting Rook Ceph upgrade user should make sure that their Kubernetes cluster is in healthy state.